src/main/java/org/terasology/caves/CaveFacetProvider.java

```diff
@@ -35,233 +36,36 @@
-        SubSampledNoise fadeCaveNoise = new SubSampledNoise(baseFadeCaveNoise, new Vector3f(0.006f * (1f / width), 0.006f * (1f / height), 0.006f * (1f / width)), 1);
         CaveFacet facet = new CaveFacet(region.getRegion(), region.getBorderForFacet(CaveFacet.class));
-        SurfaceHeightFacet surfaceHeightFacet = region.getRegionFacet(SurfaceHeightFacet.class);
 
         // get noise in batch for performance reasons.  Getting it by individual position takes 10 times as long
-        float[] caveNoiseValues = caveNoise.noise(facet.getWorldRegion());
-        float[] fadeCaveNoiseValues = fadeCaveNoise.noise(facet.getWorldRegion());
+        float[][] caveNoiseValues = new float[][]{caveNoise[0].noise(facet.getWorldRegion()),caveNoise[1].noise(facet.getWorldRegion())};
 
         for (Vector3i pos : region.getRegion()) {
-            float depth = surfaceHeightFacet.getWorld(pos.x, pos.z) - pos.y;
-            if (depth > minDepth) {
-                float noiseValue = caveNoiseValues[facet.getWorldIndex(pos)];
-                // fade caves out as they reach the surface or above the surface
-                float fadeForSurfaceCutoff = Math.min(1f - amountOfCavesNearSurface, Math.max(0f, 1f - (depth / sharpSurfaceCutoffDepth)));
-                // gradually decrease caves as they get closer to the surface
-                float fadeForScale = Math.max(0f, 1f - (depth / gradualIncreaseOverDepth)) * (1f - amountOfCavesNearSurface);
-
-                float noiseLevelIncrease = (1f - noiseLevel)
-                        * (
-                        Math.max(fadeForSurfaceCutoff, fadeForScale)
-                                // fade caves on a broad scale to stop them from being uniform
-                                // Amount added to the noise value: 1 = prevent all caves.  0 = allow normal perlin.  -1 = all caves
-                                + Math.max(0f, Math.abs(fadeCaveNoiseValues[facet.getWorldIndex(pos)]) + (2f * (1f - amountOfCaves)) - 1f)
-                );
+            int i = facet.getWorldIndex(pos);
+            float noiseValue = (float) Math.hypot(caveNoiseValues[0][i], caveNoiseValues[1][i]);
 
-                facet.setWorld(pos, noiseValue > noiseLevel + noiseLevelIncrease);
-            }
+            facet.setWorld(pos, noiseValue < 0.2);
         }
 
         region.setRegionFacet(CaveFacet.class, facet);
     }
-
-
-    @Override
-    public String getConfigurationName() {
-        return "Caves";
-    }
-
-    @Override
-    public Component getConfiguration() {
-        return configuration;
-    }
-
-    @Override
-    public void setConfiguration(Component configuration) {
-        this.configuration = (CaveFacetProviderConfiguration) configuration;
-    }
-
-    private static class CaveFacetProviderConfiguration implements Component {
```

Comment:
configuration is currently not working anyways. I think it makes sense to bring back the options one by one **as soon as the configuration is fixed**.

Reply:
This is fine. It doesn't actually need any edits now though, right?

Reply:
right, removing this alltogether is fine.